hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -96,6 +105,43 @@ public class TestHiveCatalog extends HiveMetastoreTest {
 
   @TempDir private Path temp;
 
+  private HiveMetastoreSetup hiveMetastoreSetup;
+
+  private static final String HIVE_DB_NAME = "hivedb";
```

Comment:
why not just call this `DB_NAME` so that you don't have to adjust all of the places?